### DISCUSSION TITLE
illa builder权限控制

### DISCUSSION THREAD

**Discussion by mcfee-618** [illacloud/illa-builder#1335] [General]
目前本地化部署是OK的，希望这个项目能支持多账号并且支持管理员配置不同账号的对于生成链接的访问权限控制【目前好像不支持】，并且支持用户名密码登录的方式。

**Comment by wenzhuo2022** [comment]
您好，感谢反馈。细分权限是打算支持的，不过不在近期的计划里。能了解一下希望用 用户名密码登录的原因吗

**Reply by mcfee-618** [reply]
本地使用我们一般不依赖邮箱的，主要还是依赖的账号密码。
